### DISCUSSION TITLE
How you can mod it

### DISCUSSION THREAD

**Discussion by BambiGaming2022** [DEMOLITIONDON96/FE-Feather-Custom#9] [Q&A]
How you can mod this engine?

**Comment by ThatOneSillyGuyDon** [comment]
This engine uses HScript for modding, you can view pre-existing files in your download of the engine to see how certain scripts work!
The Wiki is also a place you can go to get anymore info, which is gonna be updated here soon, since there's a lot of new features that the wiki has no info on atm

basically, everything is currently modifiable but the menus atm

however, if you don't wanna script a new character, you can always use Psych character jsons as an alterative!

for the more detailed info, I'll be adding it to the wiki soon

**Comment by Theoyeah** [comment]
how is this place alive
